frontend/javascripts/oxalis/view/right-border-tabs/connectome_tab/connectome_view.js

```diff
@@ -0,0 +1,784 @@
+
+    return isConnectomeMappingActive ? null : (
+      <Alert
+        message={
+          <>
+            The mapping this connectome was computed for is not active.{" "}
+            <a href="#" onClick={() => this.activateConnectomeMapping()}>
+              Click to activate.
+            </a>
+          </>
+        }
+        type="info"
+        showIcon
+        style={{ marginBottom: 10 }}
+      />
+    );
+  }
+
+  getConnectomeHeader() {
+    const { activeAgglomerateIds, segmentationLayer, currentConnectomeFile } = this.props;
+    const { availableSynapseTypes, connectomeData } = this.state;
+    const activeAgglomerateIdString = activeAgglomerateIds.length
+      ? activeAgglomerateIds.join(",")
+      : "";
+
+    const disabled = currentConnectomeFile == null;
+
+    return (
+      <>
+        <Input.Group compact className="compact-icons" style={{ marginBottom: 10 }}>
+          <Tooltip title="Show Synaptic Connections for Segment ID(s)">
+            <InputComponent
+              value={activeAgglomerateIdString}
+              onPressEnter={this.handleChangeActiveSegment}
+              placeholder="Segment ID 1[, Segment ID 2, ...]"
+              style={{ width: 220 }}
+              disabled={disabled}
+            />
+          </Tooltip>
+          <ButtonComponent onClick={() => this.reset()} disabled={disabled}>
+            Reset
+          </ButtonComponent>
+          <ConnectomeFilters
+            availableSynapseTypes={availableSynapseTypes}
+            connectomeData={connectomeData}
+            onUpdateFilteredConnectomeData={this.onUpdateFilteredConnectomeData}
+            disabled={disabled}
+          />
+          <ConnectomeSettings segmentationLayer={segmentationLayer} />
+        </Input.Group>
+        {this.getConnectomeMappingActivationAlert()}
+      </>
+    );
+  }
+
+  getSynapseTree() {
+    const { activeAgglomerateIds, currentConnectomeFile } = this.props;
+    const { filteredConnectomeData, checkedKeys, expandedKeys } = this.state;
+
+    if (currentConnectomeFile == null) {
+      return <Empty image={Empty.PRESENTED_IMAGE_SIMPLE} description="No connectome available." />;
+    } else if (activeAgglomerateIds.length === 0 || filteredConnectomeData == null) {
+      return (
+        <Empty
+          image={Empty.PRESENTED_IMAGE_SIMPLE}
+          description="No segment selected. Use the input field above to enter a segment ID."
+        />
+      );
+    } else {
+      return (
+        <SynapseTree
+          checkedKeys={checkedKeys}
+          expandedKeys={expandedKeys}
+          onCheck={this.handleCheck}
+          onExpand={this.handleExpand}
+          onChangeActiveAgglomerateIds={this.setActiveConnectomeAgglomerateIds}
+          connectomeData={filteredConnectomeData}
+        />
+      );
+    }
+  }
+
+  render() {
+    const { segmentationLayer } = this.props;
+
+    // Render the tab in the background to avoid rebuilding the tree when switching tabs.
```

Comment:
Does this comment refer to a particular line of code here? I thought, this is already covered by https://github.com/scalableminds/webknossos/pull/5894/files#diff-8062b2d313f68c910f76cce564fa2a4af33d48232c4ea9e79ff3c707da0e33caR148. If so, I'd remove the comment here.